drools-benchmarks/pom.xml

```diff
@@ -61,6 +61,16 @@
       <groupId>org.kie</groupId>
       <artifactId>kie-pmml-dependencies</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-api</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.optaplanner</groupId>
+          <artifactId>optaplanner-core</artifactId>
```

Comment:
I'm not understanding why do we need to add optaplanner, is it necessary?

Reply:
@mariofusco optaplanner is excluded here. `kie-server-api` depends on `optaplanner-core` but it causes a kie.conf problem when creating an jmh uber jar. So excluded.

Reply:
In other words, optaplanner7 has to apply the kie.conf namespace approach?

Reply:
Ah ok, sorry I misread this.